app/frontend/public/store/index.html

```diff
@@ -38,7 +39,7 @@
 
         <section id="home">
             <img src="img/ape1.jpg" alt="ape" />
-            <button class="order"><b>Order Now</b></button>
+            <button id="order" class="order"><b>Order Now</b></button>
```

Comment:
For creating a button to link to the ordering page with the blue click style, you can do this:

```suggestion
            <a href="store/ordering">
                <div id="order btn btn-primary" class="order">
                    <b>Order Now</b>
                </div>
            </a>
```

Reply:
in style.css in the same folder, you can replace `#home button` with `order`

then you can remove webPage.js as it wont be needed

Reply:
okay changed it!!